main/src/library/File.flix

```diff
@@ -15,6 +15,57 @@
  */
 namespace File {
 
+    ///
+    /// File types.
+    ///
+    enum FileType with Eq, Order, ToString {
+        case File,
+        case Directory,
+        case SymLink,
+        case Other
+    }
+
+    ///
+    /// Returns the access time of the given file `f` in milliseconds.
+    ///
+    pub def accessTime(f: String): Result[Int64, String] = ???
```

Comment:
Is this supposed to be `the last access time`?

Reply:
I think so